python/sparkdl/graph/builder.py

```diff
@@ -75,6 +78,48 @@ def run(self, *args, **kargs):
         """
         return self.sess.run(*args, **kargs)
 
+    def asUDF(self, udf_name, fetches, feed_dict=None, blocked=False, register=True):
+        """ 
+        Make the TensorFlow graph as a SQL UDF 
+
+        :param udf_name: str, name of the SQL UDF 
+        :param fetches: list, output tensors of the graph
+        :param feed_dict: dict, a dictionary that maps graph elements to input values
+        :param blocked: bool, whether the TensorFrame execution should be blocked based or row based
+        :param register: bool, whether this UDF should be registered
+        :return: JVM function handle object
```

Comment:
this return value is very unhelpful for python users, but I want to propose a way to wrap a scala UDF automatically. No need to change it for now.

Update the doc though to mention that it is of class `... UserDefinedFunction`